pyspecProcScripts/phasing.py

```diff
@@ -358,8 +357,10 @@ def hermitian_function_test(
         fl.twinx(orig=False, color="red")
         residual.name("cost function")
         fl.plot(residual, color="r", alpha=0.5, human_units=False)
-        ylim(0, residual["center" : (best_shift - 4e-3, best_shift)].data.max())
-        axvline(x=best_shift, c="k", linestyle=":")
+        fl.plot(residual["center" : (best_shift - 4e-3, best_shift)], ':', alpha=0.5, human_units=False)
```

Comment:
it was hard to distinguish the best shift from the other lines, so I changed the plotting style.